### PR TITLE
(DOCSP-11274): Android Quickstart Output

### PR DESCRIPTION
**Pull request by mongomoe**
jira:
---
* https://jira.mongodb.org/browse/DOCSP-11274

stage:
------
* https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-11274-Quickstart-Output-Android-SDK/android/quick-start.html#complete-example

### CODE REVIEW

File: source/android/quick-start.txt [hunk 1]
```diff
@@ -366,3 +366,186 @@ Once logged in, you can log out:
...
+         import io.realm.Realm
+         import io.realm.mongodb.App
+         import io.realm.mongodb.AppConfiguration
+         import io.realm.RealmObject
+         import io.realm.annotations.PrimaryKey
+         import io.realm.annotations.Required
+         import io.realm.mongodb.Credentials
+         import io.realm.mongodb.User
+         import io.realm.mongodb.sync.SyncConfiguration
+         import org.bson.types.ObjectId
```
Comment (nathan-contino-mongo) [inline source/android/quick-start.txt:474]: I always find that grouping imports like you've done with the Java example tends to make them easier to read. Can we organize these by package?

File: source/android/quick-start.txt [hunk 2]
```diff
@@ -366,3 +366,186 @@ Once logged in, you can log out:
...
+         }
+
+
+Output
+------
+Running the above code should output something like the following:
+
+.. code-block:: shell
+
+   Successfully authenticated anonymously
```
Comment (nathan-contino-mongo) [inline source/android/quick-start.txt:547]: It looks like the Kotlin example contains a period after "anonymously", but the Java example doesn't. Can we add a period to the Java example and here?

File: source/android/quick-start.txt [hunk 3]
```diff
@@ -366,3 +366,186 @@ Once logged in, you can log out:
...
+               Realm.getInstanceAsync(config, object: Realm.Callback() {
+                     override fun onSuccess(_realm: Realm) {
+                        // since this realm should live exactly as long as this activity, assign the realm to a member variable
+                        realm = _realm
+                        Log.v("QUICKSTART", "Successfully instantiated realm!");
+                     }
+               })
+
+               user?.logOutAsync {
+                     if (it.isSuccess) {
```
Comment (nathan-contino-mongo) [inline source/android/quick-start.txt:521]: It looks like all of the lambdas in both of these examples have an extra level of indentation. Can we remove the extra level for consistency with the example snippets above?

Comment (mongomoe) [reply source/android/quick-start.txt]: 👍

File: source/android/quick-start.txt (hunk 2 above)
Comment (mongomoe) [reply source/android/quick-start.txt]: 👍

File: source/android/quick-start.txt (hunk 1 above)
Comment (mongomoe) [reply source/android/quick-start.txt]: 👍

**Pull request merged**